test/functional/rpc_fundrawtransaction.py

```diff
@@ -1143,6 +1144,21 @@ def test_feerate_rounding(self):
         rawtx = w.createrawtransaction(inputs=[], outputs=[{self.nodes[0].getnewaddress(address_type="bech32"): 1 - 0.00000202}])
         assert_raises_rpc_error(-4, "Insufficient funds", w.fundrawtransaction, rawtx, {"fee_rate": 1.85})
 
+    def test_sffo_with_negative_ev(self):
+        self.log.info("Testing that subtract fee from outputs with negative effective value outputs still works")
```

Comment:
```suggestion
    def test_wallet_sweaping(self):
        self.log.info("Testing that wallet sweeping works even with negative effective value.")
```

Reply:
the functionality change isn't for sweeping per-se, maybe just add "often used for sweeping" in the comment

Reply:
I am finding this line a little a confusing, maybe something like "Testing that sweeping transaction with "subtract fee from outputs" with negative effective value utxos works".

Reply:
This functionality is for more than just sweeping. I've changed the text to `Test that enable subtractFeeFromOutputs will include negative effective value UTXOs`.